src/basecamp.cpp

```diff
@@ -164,10 +164,9 @@ void basecamp::add_expansion( const std::string &bldg, const tripoint &new_pos,
     update_resources( bldg );
 }
 
-void basecamp::define_camp( npc &p, const std::string &camp_type )
+void basecamp::define_camp( const tripoint omt_pos, const std::string &camp_type )
 {
     query_new_name();
-    omt_pos = p.global_omt_location();
```

Comment:
`omt_pos` here is actually a member of `basecamp`, so you should still assign the argument value to it.

Reply:
Ah gotcha.  Could I do something like `self.omt_pos = omt_pos` or should I rename the argument in the signature variable and do `omt_pos = p` or whatever?

Reply:
the latter.